fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java

```diff
@@ -99,6 +99,8 @@
  */
 public class StatsCalculator extends DefaultPlanVisitor<StatsDeriveResult, Void> {
 
+    public static final StatsCalculator STATS_CALCULATOR = new StatsCalculator(null);
```

Comment:
what's STATS_CALCULATOR used for?

Reply:
Calculate cost for plan without groupexpression